cli/ops/os.rs

```diff
@@ -57,18 +58,42 @@ fn op_start(
   })))
 }
 
-fn op_home_dir(
+#[derive(Deserialize)]
+struct GetDirArgs {
+  name: std::string::String,
+}
+
+fn op_get_dir(
   state: &ThreadSafeState,
-  _args: Value,
+  args: Value,
   _zero_copy: Option<PinnedBuf>,
 ) -> Result<JsonOp, ErrBox> {
   state.check_env()?;
-  let path = dirs::home_dir()
+  let args: GetDirArgs = serde_json::from_value(args)?;
+
+  let path = match args.name.as_str() {
+    "home" => dirs::home_dir(),
+    "config" => dirs::config_dir(),
+    "cache" => dirs::cache_dir(),
+    "data" => dirs::data_dir(),
+    "data_local" => dirs::data_local_dir(),
+    "audio" => dirs::audio_dir(),
+    "desktop" => dirs::desktop_dir(),
+    "document" => dirs::document_dir(),
+    "download" => dirs::download_dir(),
+    "font" => dirs::font_dir(),
+    "picture" => dirs::picture_dir(),
+    "public" => dirs::public_dir(),
+    "template" => dirs::template_dir(),
+    "video" => dirs::video_dir(),
+    _ => None,
```

Comment:
Is this case tested?

Reply:
No test cases needed here.

I change this panic when not match.

So if not match case, an exception will be thrown and the CI won't pass

Reply:
I think it should be tested to confirm it returns an error (not panics)

Reply:
Now it returns an error

But I'm not sure where this test case should be written

Maybe `cli/js/os_test.ts` or` cli/js/dispatch_json.ts`?